plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.ts

```diff
@@ -63,6 +66,24 @@ export function createFetchCookiecutterAction(options: {
             description: 'Values to pass on to cookiecutter for templating',
             type: 'object',
           },
+          copyWithoutRender: {
+            title: 'Copy Without Render',
+            description:
+              'Avoid rendering directories and files in the template',
+            type: 'object',
```

Comment:
Think this should be:

```ts
{
  type: 'array',
  items: {
    type: 'string'
  }
}
```

Reply:
aah, didn't know about that, i've updated. cheers.